src/components/CategoryList.js

```diff
@@ -54,7 +57,7 @@ export class CategoryList extends React.PureComponent {
             section={section}
           />
         )}
-        renderSectionHeader={this.renderSectionHeader}
+        renderSectionHeader={noSectionHeader ? null : this.renderSectionHeader}
```

Comment:
let's put the condition inside of `renderSectionHeader` to have the render part more readable here and the logic separated.

example for update of the function:

```
const { noSectionHeader } = this.props;

if (!data?.length || noSectionHeader) return null;
```

Reply:
Yes I did and it worked fine 👍